blips/blip-0001.md

```diff
@@ -0,0 +1,289 @@
+mailing list](https://lists.linuxfoundation.org/mailman/listinfo/lightning-dev). This 
+gives the author a chance to flesh out the draft bLIP to make it properly formatted, 
+of high quality, and to address additional concerns about the proposal. Following a 
+discussion, the proposal should be submitted to the [bLIPs folder of the lightning-rfc
+repository](https://github.com/lightningnetwork/lightning-rfc) as a pull request. This 
+draft must be written in bLIP style as described below, and its bLIP number will be 
+the PR number automatically assigned by Github (or, if preferred by the author, the 
+Issue # if there was discussion in the Issues section of this repository about this bLIP).
+
+When the bLIP draft is complete, the lightning-rfc editors will check the requested 
+Feature Bit, Message Type, and/or TLV assignments for collisions. If there are no 
+issues, the lightning-rfc editors will merge the pull request into the [bLIPs folder 
+of the lightning-rfc repository](https://github.com/lightningnetwork/lightning-rfc). 
+The editors will not unreasonably reject a bLIP. Reasons for rejecting bLIPs include 
+duplication of effort, disregard for formatting rules, being too unfocused or too 
+broad, being technically unsound, not providing proper motivation or addressing 
+backwards compatibility, or not in keeping with the Bitcoin and Lightning Network 
+philosophy. For a bLIP to be accepted it must meet certain minimum criteria. It 
+must be a clear and complete description of the proposed enhancement. The enhancement must 
+represent a net improvement. The proposed implementation, if applicable, must be solid 
+and must not complicate the protocol unduly.
+
+The bLIP author may update the draft as necessary in the git repository. Updates to 
+drafts should also be submitted by the author as pull requests.
+
+## Transferring bLIP Ownership
+
+It occasionally becomes necessary to transfer ownership of bLIPs to a new champion. In 
+general, we'd like to retain the original author as a co-author of the transferred bLIP, 
+but that's really up to the original author. A good reason to transfer ownership is 
+because the original author no longer has the time or interest in updating it or 
+following through with the bLIP process, or has fallen off the face of the 'net (i.e. is 
+unreachable or not responding to email). A bad reason to transfer ownership is because 
+you don't agree with the direction of the bLIP. We try to build consensus around a bLIP, 
+but if that's not possible, you can always submit a competing bLIP.
+
+If you are interested in assuming ownership of a bLIP, send a message asking to take over, 
+addressed to both the original author and the bLIP editor. If the original author doesn't 
+respond to email in a timely manner, the bLIP editor will make a unilateral decision (it's 
+not like such decisions can't be reversed).
+
+### bLIP Editors
+
+The current BOLT editors are the bLIP editors.
```

Comment:
Maybe worth listing explicitly as this list will evolve over time?

Reply:
Listed explicitly, hope I'm not missing anybody 🙈